pyproject.toml

```diff
@@ -64,15 +65,21 @@ boto3-stubs = "^1.34.67"
 streamlit = "^1.32.2"
 lxml = "^5.1.0"
 loguru = "^0.7.2"
-streamlit-feedback = "^0.1.3"
-html2markdown = "^0.1.7"
 
 [tool.poetry.group.ai.dependencies]
 anthropic = "^0.21.1"
 litellm = "^1.32.7"
 openai = "^1.14.2"
 
 
+[tool.poetry.group.streamlitapp.dependencies]
+cognitojwt = "^1.4.1"
```

Comment:
Any need for `pandas` or `cognitojwt` in the other poetry group?

Reply:
no, this is a merge error. i will remove it

Reply:
[1d83a24](https://github.com/i-dot-ai/redbox-copilot/pull/134/commits/1d83a24ea5c70d162aaeecb7a8ee49f5e464757f)